cmd/seccat/seccat_plan9.go

```diff
@@ -0,0 +1,8 @@
+package main
```

Comment:
Seems like we should add `!build plan9` here to be safe (e.g. in case someone change seccat_posix to be more OS specific)

Reply:
`+build plan9` is redundant because filenames ending with `_plan9.go` is only built on plan9. I just checked all the `_windows.go` files and only 1 of the 4 files have `+build windows`, so we usually don't add a redundant build tag.  Am I missing something?

Reply:
Fair point you can leave it as is